client/cucumber/features/local-fabric-ops.feature

```diff
@@ -1,8 +1,17 @@
 Feature: Local Fabric Ops
     Tests all the features of the fabric ops panel
 
-    Scenario Outline: There should be a tree item (started)
+    Scenario Outline: There should be a tree item (disconnected)
         Given the Local Fabric is running
+        Then there should be a tree item with a label '<label>' in the 'Local Fabric Ops' panel
+        And the tree item should have a tooltip equal to '<tooltip>'
+        Examples:
+        | label                  | tooltip                |
+        | Local Fabric  ●        | local_fabric           |
```

Comment:
I know this is a cucumber file, but can we change the names so that "Local Fabric" and "local_fabric" match please?